### DISCUSSION TITLE
https://imagekit.io/blog/nextjs-video-player/

### DISCUSSION THREAD

**Discussion by giscus** [imagekitio/discussions#34] [Blogs]
# https://imagekit.io/blog/nextjs-video-player/

This blog'll demonstrate how to integrate a video player into a Next.js application. We'll examine the <video> tag's features and limitations and introduce the video.js library to enable advanced functionalities, such as Adaptive Bitrate Streaming (ABS).

https://imagekit.io/blog/nextjs-video-player/

**Comment by haniframadhani** [comment]
what the different between default video in next js and video js?
